### DISCUSSION TITLE
Can I create an inbox between Chatwoot and Facebook by API?

### DISCUSSION THREAD

**Discussion by hoaibao256** [chatwoot/chatwoot#5489] [Chatwoot APIs]
I want to create an inbox from Chatwoot to Facebook by using API code.
Specifically I want to use API code instead of 4 steps below.
Please help!
Thanks!
![image](https://user-images.githubusercontent.com/83713864/191914977-939f39df-8b62-434c-b5e0-a1878eb6dd9f.png)

**Comment by tejaswinichile** [comment]
@hoaibao256 We can not create a Facebook channel inbox with the API version, as you need to log in with the Facebook dialogue box provided.
